rector.php

```diff
@@ -26,4 +26,6 @@
         ClosureToArrowFunctionRector::class,
         RemoveExtraParametersRector::class => [__DIR__ . '/src/Service/Generate/PhpRenderer.php'],
     ]);
+
+    $rectorConfig->disableParallel();
```

Comment:
Is there a problem? Is this really necessary?

Reply:
If it is not disabled, the rector fails in version 0.17.